Dockerfile

```diff
@@ -47,6 +47,9 @@ RUN npm install
 # add the rest of the app files
 ADD --chown=snyk:snyk . .
 
+# OpenShift 4 doesn't allow the user can't access the app files without giving permission access.
+RUN chmod +x .
```

Comment:
1. if possible to improve the syntax of the comment :) 
2. why do you need the +x? what is it trying to execute? I wonder if +r or +w should be enough
3. don't you need to make that recursive? I find it weird that it works 😆

Reply:
pay attention that your chmod +x patch only applies to the app folder, I wonder if for some reason chmod +x bin will be enough

Reply:
The problem was that `dumb-init` was trying to access the `bin/start` and it didn't have permission. I tried to be more specific to access only `bin/start` and it didn't work. But let me try your suggestions, I prefer it would be more specific than generic too.